### PR TITLE
config/dtb: bq-piccolo: Add truly-nt35695 panel driver

### PR DESCRIPTION
**Pull request by a-andre**

### CODE REVIEW

File: config/bq-piccolo.sh [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+# DTB: From https://dl.twrp.me/piccolo/twrp-3.7.0_9-0-piccolo.img
+
+OPTIONS=(-r power)
+PANELS=(
+	[truly_nt35695_1080p_video]="longcheer,truly-nt35695"
```
Comment (stephan-gh) [inline config/bq-piccolo.sh:6]: iirc this panel driver had the problem that some pixel lines at the top of the screen are missing. Is this still the case? Iirc I fixed this somewhat by using the `_cmd` variant, changing it to video mode and messing with some timings.

Comment (a-andre) [reply config/bq-piccolo.sh:6]: I don't know if the lines are still missing.

The "default" `_cmd` variant causes artefacts/flickering, that's why I would like to keep using the `_video` variant for now.

Comment (stephan-gh) [reply config/bq-piccolo.sh:6]: Can you check again by viewing https://matrix.org/_matrix/media/r0/download/postmarketos.org/dBXSFrCgAeveNwJDALcCtnpq full screen on the phone? The top green line might be missing.

Comment (a-andre) [reply config/bq-piccolo.sh:6]: The top green line is missing.

Comment (stephan-gh) [reply config/bq-piccolo.sh:6]: Can you try this panel driver: https://github.com/msm8916-mainline/linux/commits/tmp/piccolo-panel

It's the command mode one, switched to video mode plus a potential fix for the green line. Please test without the "pixel line" commit first and see if that already fixes the green line problem.

This is taken from my lk1st panel driver, I haven't tested it on Linux so far.

Comment (a-andre) [reply config/bq-piccolo.sh:6]: I have to apply both commits to fix the green line problem completely.
If only https://github.com/msm8916-mainline/linux/commit/ac4c7b49d989186076daaa6324b350f2af497b4b is applied, the green line is visible at the top if the device has normal orientation. But when the device is rotated by 180° (bottom-up orientation), the green line moves to the bottom of the image.

Comment (stephan-gh) [reply config/bq-piccolo.sh:6]: Sounds good, can you integrate these changes into yours?

Comment (a-andre) [reply config/bq-piccolo.sh:6]: You mean, add the two commits to https://github.com/msm8916-mainline/linux/pull/251?
If yes, how? Should I add a commit with the generated panel driver first, so your fixes can be applied?

Comment (stephan-gh) [reply config/bq-piccolo.sh:6]: > Should I add a commit with the generated panel driver first, so your fixes can be applied?

Exactly, yes

File: config/bq-piccolo.sh [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+# DTB: From https://dl.twrp.me/piccolo/twrp-3.7.0_9-0-piccolo.img
+
+OPTIONS=(-r power --backlight-gpio)
```
Comment (stephan-gh) [inline config/bq-piccolo.sh:4]: Please add `--dumb-dcs` so the panel driver doesn't look so incredibly weird (also I think the generator warns about this). It doesn't look like the panel driver uses DCS according to the specification.

**Comment by stephan-gh** [comment]
Pushed, thanks!

**Pull request closed without merging**